packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -322,6 +322,9 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
         return defaultOnRender(inlineImage);
       }
 
+      const pointerEvents =
+        inlineImage.imageAttributes.src === '' || inlineImage.imageAttributes.src === 'blob://' ? 'none' : 'unset';
```

Comment:
`inlineImage.imageAttributes.src === 'blob://'`
Instead of checking for this, recommend we check against the error object in resource cache.

Reply:
That way in the future if we want to display a pop up with the error message we can leverage that here